src/main/resources/com/mathworks/ci/UseMatlabVersionBuildWrapper/help-matlabRootFolder.html

```diff
@@ -1,8 +1,9 @@
 <div>
-  Enter the full path to the MATLAB root folder, which is returned by the <i><b>matlabroot</b></i> function. The specified directory will be prepended to the PATH environment variable, and the specified MATLAB version will be used to perform the build.
-  You must not leave this field blank if no MATLAB directory exists on the path. If you leave the field blank, then the system tries to use the MATLAB directory closest to the top of the system path. <br><br><br>Example:<br><br><i><b><u>Windows:</u></b></i><br> C:\Program Files\MATLAB\R2019a <br>
-  <i><b><u>Linux:</u></b></i><br> /usr/local/MATLAB/R2019a <br>
-  <i><b><u>Mac:</u></b></i><br> /Applications/MATLAB_R2019a.app <br>
+  Enter the full path to the MATLAB root folder, which is returned by the <i><b>matlabroot</b></i> function. The plugin prepends the folder to the PATH environment variable and uses the specified MATLAB version to perform the build.
+  You must not leave this box blank if no MATLAB folder exists on the path. If you leave it blank, the plugin attempts to use the topmost MATLAB folder on the system path.
+  <br><br><br>Here are examples of the root folder path on different platforms:<br><br><i><b><u>Windows:</u></b></i><br> C:\Program Files\MATLAB\R2020b <br>
+  <i><b><u>Linux:</u></b></i><br> /usr/local/MATLAB/R2020b <br>
+  <i><b><u>macOS:</u></b></i><br> /Applications/MATLAB_R2020b.app <br>
```

Comment:
The same for the platform names. Do we have to underline, italicize, and bold them at the same time?

Reply:
Done. Changed to just bold.